### ISSUE TITLE
Can't use TF modules from registry

### ISSUE THREAD

**Issue by gugaiz** [scalefactory/tfctl#12]
I created my own profile called `vpc-profile` that uses the terraform vpc module

#### **`main.tf`**
``` 
module "vpc" {
  source  = "terraform-aws-modules/vpc/aws"
  version = "2.44.0"
...
```

when I run `tfctl --all -d -- init` I get (twice)

```
info: account-dev: Initializing modules...
info: account-dev: Downloading terraform-aws-modules/vpc/aws 2.44.0 for vpc-profile.vpc...
error: account-dev: 
error: account-dev: Error: Failed to download module
error: account-dev: 
error: account-dev: Could not download module "vpc" (../../../profiles/vpc-profile/main.tf:10)
error: account-dev: source code from
error: account-dev: "git::https://github.com/terraform-aws-modules/terraform-aws-vpc?ref=v2.44.0":
error: account-dev: error downloading
error: account-dev: 'https://github.com/terraform-aws-modules/terraform-aws-vpc?ref=v2.44.0': git 
error: account-dev: must be available and on the PATH.
error: account-dev: 
error: account-dev: 
```
It is like it is not finding git command, which I obviously have installed on my computer.

```
$ which git
/usr/bin/git
```

**Comment by gugaiz** [comment]
I think I found the issue. The problem is that you are getting rid of all env variables hen calling `ENV.keep_if` [here](https://github.com/scalefactory/tfctl/blob/master/lib/tfctl/executor.rb#L52), then terraform command does not have the correct env setup. PR #13 solves the problem for me

**Comment by word** [comment]
Thanks for the bug report.  This should be fixed in the latest release: 1.2.2.  Details in PR #13.
